### PR TITLE
Added an HTML target attribute to the contributors github link 

### LINKED ISSUE #57: The GitHub link icon on the contributor's profile should Open the linked document in a new window or tab

### PR DESCRIPTION
**Pull request by Godwindaniel10**
@Signior-X 
Fixes #57

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
     "editor.formatOnSave": true,
-    "editor.tabSize": 2
+    "editor.tabSize": 2,
+    "workbench.colorCustomizations": {
```
Comment (Signior-X) [inline .vscode/settings.json:4]: Undo this change.

Try to never have any unnecessary changes in a PR, and only the changes needed.

File: contributors.html [hunk 2]
```diff
@@ -137,7 +137,7 @@ <h3>Links</h3>
                 <h3> ` + element.login + ` </h3>
                 <p> ` + element.contributions + ` commits </p>
                 <ul class="probootstrap-footer-social">
-                  <li class="facebook"><a href="` + element.html_url + `"><i
+                  <li class="facebook"><a href="` + element.html_url + `" target="_blank"><i
```
Comment (Signior-X) [inline contributors.html:140]: > rel="noopener noreferrer" should be added to links containing target="_blank" as a precaution against reverse tabnabbing


So also add `rel="noopener noreferrer"` to this line.

**Review by Signior-X — CHANGES_REQUESTED** [review]
Just make this few changes and this will be ready to merge.

File: contributors.html (hunk 2 above)
Comment (Godwindaniel10) [reply contributors.html:140]: ok

**Comment by Godwindaniel10** [comment]
changes made.      i revert ( settings.json ) file, and added the rel="noopener noreferrer" to the link

**Review by Signior-X — APPROVED** [review]
Perfect! Thanks for this one @Godwindaniel10 
Great and the PR the way you created is good.

**Comment by Godwindaniel10** [comment]
@Signior-X thanks 😊👍
